packages/common/src/static.ts

```diff
@@ -1,9 +1,14 @@
 export const ONION_ADDRESS_REGEX = /^[a-z0-9]{56}$/g
+export const PEER_ID_REGEX = /^[a-zA-Z0-9]{46}$/g
+export const PSK_LENGTH = 44 // PSK is 256 bits/8 = 32 bytes which encodes to 44 characters base64
```

Comment:
Because it's used in function that is used across few packages.

Reply:
libp2p.ts is also in the common package. This is totally a nitpick and we probably shouldn't spend too much time on this, but in general I think it's easier to read if libp2p related things are together in a single or a few files, like libp2p.ts and libp2p.service.ts. I don't see the benefit of splitting things up with this level of granularity where we put all static variables in a single file - at least not right now. I think it can make it more difficult to read because it's another file that we need to jump to when reading the code.